### PR TITLE
New Files Added: Assingment7 and assingment8

### PR DESCRIPTION
**Pull request by vinod2406**

### CODE REVIEW

File: vinod/html/ASSIN7/job.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" type="text/css" href="job1.css" />
+        <title>home page</title>
+    </head>
+    <body>
+        <nav class="navigation">
+            <ul class="nav-list">
+                <li><a href="login.html">Login</a></li>
+
```
Comment (shivling123) [inline vinod/html/ASSIN7/job.html]: remove space , extraline

File: vinod/html/ASSIN7/login.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    </head>
+    <body>
+        <div class="block">
+            <div class="block1">
+                <h2>Login Form</h2>
+            </div>
+            <div class="block2">
+                <label for="uname"><b>Username</b></label>
+                <input type="text" placeholder="Enter Username" name="uname" required /><br /><br />
+
```
Comment (shivling123) [inline vinod/html/ASSIN7/login.html]: remove space , extraline

**Review by akash19972306 — APPROVED** [review]

**Review by shivling123 — CHANGES_REQUESTED** [review]
remove extra spaces in code and
push again asasp

**Review by Pranjali1999 — APPROVED** [review]

**Review by Akash-ing — APPROVED** [review]

File: vinod/html/ASSIN7/job.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    </head>
+    <body>
+        <nav class="navigation">
+            <ul class="nav-list">
+                <li><a href="login.html">Login</a></li>
+                <li><a href="job.html">home</a></li>
+            </ul>
+            <div>
+                <center><h1>job portal</h1></center>
+                <img src="job2.jpeg"style="height:600px; width:1300px;alt="job">
```
Comment (SnehaMakade) [inline vinod/html/ASSIN7/job.html]: there should be space after colon always

File: vinod/html/ASSIN7/job1.css [hunk 4]
```diff
@@ -0,0 +1,74 @@
+   ul {
```
Comment (SnehaMakade) [inline vinod/html/ASSIN7/job1.css:1]: always add a space after colon

File: vinod/html/ASSIN7/job1.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    }
+   li a:hover {
+      background-color: red ;
+    }
+   body{
+        background-color: aquamarine;
+    }
+   body {font-family:Arial,Helvetica,sans-serif;}
+   form {border:3px solid #f1f1f1;}
+   input[type=text],input[type=password] {
```
Comment (SnehaMakade) [inline vinod/html/ASSIN7/job1.css:26]: always add a space after comma and colon

File: vinod/html/ASSIN7/presonal.html [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+            <input type="checkbox" id="Oracal" name="Skill Set" value="Oracal"/>
+            <label for="Oracal"> Oracal</label>
+            </div><br>
+            <div class="btn">
+                <button type="button" class="continue"><a href="professional.html">Continue</a></button>
+            </div>
+        </form>
+      </block>
+    </body>
+</html>
```
Comment (SnehaMakade) [inline vinod/html/ASSIN7/presonal.html:58]: add new end line in all files

File: vinod/html/ASSIN7/professional.html [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+            <div class="block1">
+                <h1><em>Professional Details</em></h1>
+            </div>
+            <div class="block2">
+                <h3>Work Experiance</h3>
+                <lable for="company">company:</lable>
+                <input type="text"id="company"name="company"/><br><br>
+                <lable for="job title">job title:</lable>
+                <input type="text"id="job title"name="job title"/><br/><br/>
+                <lable for="description">description:</lable>
```
Comment (SnehaMakade) [inline vinod/html/ASSIN7/professional.html]: label should start with capital letter

**Review by SnehaMakade — CHANGES_REQUESTED** [review]
do changes as per comments given in all files

**Review by sourabh513 — APPROVED** [review]

**Pull request merged**